pkg/cmd/gtctl/connect/connect.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connect
+
+import (
+	"errors"
+	"github.com/GreptimeTeam/gtctl/pkg/cmd/gtctl/connect/mysql"
+	"github.com/GreptimeTeam/gtctl/pkg/logger"
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+const (
+	SplitSeparator = "://"
+	MySQL          = "mysql"
+)
+
+func NewConnectCommand(l logger.Logger) *cobra.Command {
```

Comment:
suggestion: maybe the `connect` subcommand will be better in `cluster`? For example:

```
gtctl cluster connect <cluster-name> -p mysql
```

Reply:
is it need add flag like username, password and address?Like this:
```
gtctl cluster connect <cluster-name> -p mysql -P 4001 -u root@root
```

Reply:
@sjcsjc123 yes, but it can be easier, like:

```
gtctl cluster connect <cluster-name> -p mysql
```

- The `-u` option can be ignored(because we don't fully support it);

- `-P 4001` can be inferred from cluster metadata(`GetCluster` API);